core/io/image_loader.cpp

```diff
@@ -62,7 +62,7 @@ Error ImageLoader::load_image(String p_file, Ref<Image> p_image, FileAccess *p_c
 
 		if (!loader[i]->recognize(extension))
 			continue;
-		Error err = loader[i]->load_image(p_image, f, p_force_linear);
+		Error err = loader[i]->load_image(p_image, f, p_force_linear, p_scale);
```

Comment:
@toger5 those are not really options, i understand your concerns but, currently this is the best we can do (that was actually the reduz suggestion), and it makes sense, maybe later we can refactor all the importers to remove unecessary parameters like force_linear (this is the same case as p_scale, only used on some importers).  so for now, it is better as is.